.github/workflows/publish-gh-pages.yml

```diff
@@ -0,0 +1,97 @@
+# Sample workflow for building and deploying a Jekyll site to GitHub Pages
+name: Publish site
+
+on:
```

Comment:
should we also add a scheduled trigger to periodically pick up API doc artifact updates?

Reply:
I think we're good with the gh-pages branch trigger as well as the manual deploy